{
  "uid" : "a0c95083a2ad09fa",
  "name" : "Поиск трат по дате",
  "fullName" : "test.test_app.TestSearch#test_search_by_date",
  "historyId" : "38f8009d7be4d8e20d113bf314c7b53f",
  "time" : {
    "start" : 1750752654475,
    "stop" : 1750752654475,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=137.0.7151.103)\nStacktrace:\n#0 0x560f950b1c9a <unknown>\n#1 0x560f94b576e0 <unknown>\n#2 0x560f94b4e7e7 <unknown>\n#3 0x560f94b3f699 <unknown>\n#4 0x560f94b413e0 <unknown>\n#5 0x560f94b3fa3d <unknown>\n#6 0x560f94b3f3af <unknown>\n#7 0x560f94b3f085 <unknown>\n#8 0x560f94b3ceee <unknown>\n#9 0x560f94b3d6ca <unknown>\n#10 0x560f94b5ac09 <unknown>\n#11 0x560f94bf5ed5 <unknown>\n#12 0x560f94bcee32 <unknown>\n#13 0x560f94bf52cc <unknown>\n#14 0x560f94bcec03 <unknown>\n#15 0x560f94b9b47b <unknown>\n#16 0x560f94b9c0e1 <unknown>\n#17 0x560f9507644b <unknown>\n#18 0x560f9507a37f <unknown>\n#19 0x560f9505df89 <unknown>\n#20 0x560f9507af18 <unknown>\n#21 0x560f950426df <unknown>\n#22 0x560f9509f308 <unknown>\n#23 0x560f9509f4e6 <unknown>\n#24 0x560f950b0b76 <unknown>\n#25 0x7f009da9caa4 <unknown>\n#26 0x7f009db29c3c <unknown>",
  "statusTrace" : "web_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"800d54e7fde08ccf741b00e4d6db2ebc\")>\napp_user = None\nenvs = Envs(frontend_url='http://frontend.niffler.dc', gateway_url='http://gateway.niffler.dc:8090', auth_url='http://auth.ni...stgresql+psycopg2://postgres:secret@gateway.niffler.dc:5432/niffler-userdata', soap_address='http://localhost:8089/ws')\n\n    @pytest.fixture(scope='module')\n    def niffler(web_driver: WebDriver, app_user, envs) -> Generator[Niffler, Any, None]:\n        niffler = Niffler(web_driver)\n>       niffler.login_page.go_to_niffler()\n\nconftest.py:90: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nmodel/LoginPage.py:20: in go_to_niffler\n    self.wd.get(self.base_url)\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:454: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py:429: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe8db9330e0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#24 0x560f950b0b76 \\\\u003Cunknown>\\\\n#25 0x7f009da9caa4 \\\\u003Cunknown>\\\\n#26 0x7f009db29c3c \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=137.0.7151.103)\nE       Stacktrace:\nE       #0 0x560f950b1c9a <unknown>\nE       #1 0x560f94b576e0 <unknown>\nE       #2 0x560f94b4e7e7 <unknown>\nE       #3 0x560f94b3f699 <unknown>\nE       #4 0x560f94b413e0 <unknown>\nE       #5 0x560f94b3fa3d <unknown>\nE       #6 0x560f94b3f3af <unknown>\nE       #7 0x560f94b3f085 <unknown>\nE       #8 0x560f94b3ceee <unknown>\nE       #9 0x560f94b3d6ca <unknown>\nE       #10 0x560f94b5ac09 <unknown>\nE       #11 0x560f94bf5ed5 <unknown>\nE       #12 0x560f94bcee32 <unknown>\nE       #13 0x560f94bf52cc <unknown>\nE       #14 0x560f94bcec03 <unknown>\nE       #15 0x560f94b9b47b <unknown>\nE       #16 0x560f94b9c0e1 <unknown>\nE       #17 0x560f9507644b <unknown>\nE       #18 0x560f9507a37f <unknown>\nE       #19 0x560f9505df89 <unknown>\nE       #20 0x560f9507af18 <unknown>\nE       #21 0x560f950426df <unknown>\nE       #22 0x560f9509f308 <unknown>\nE       #23 0x560f9509f4e6 <unknown>\nE       #24 0x560f950b0b76 <unknown>\nE       #25 0x7f009da9caa4 <unknown>\nE       #26 0x7f009db29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "[S] Envs",
    "time" : {
      "start" : 1750752638473,
      "stop" : 1750752638473,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "[S] App User",
    "time" : {
      "start" : 1750752638473,
      "stop" : 1750752639140,
      "duration" : 667
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Создать пользователя 'qa_guru'",
      "time" : {
        "start" : 1750752638473,
        "stop" : 1750752639140,
        "duration" : 667
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "user_name",
        "value" : "'qa_guru'"
      }, {
        "name" : "user_password",
        "value" : "'qa_guru'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "[M] Web Driver",
    "time" : {
      "start" : 1750752639891,
      "stop" : 1750752652549,
      "duration" : 12658
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "[M] Niffler",
    "time" : {
      "start" : 1750752652550,
      "stop" : 1750752653136,
      "duration" : 586
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=137.0.7151.103)\nStacktrace:\n#0 0x560f950b1c9a <unknown>\n#1 0x560f94b576e0 <unknown>\n#2 0x560f94b4e7e7 <unknown>\n#3 0x560f94b3f699 <unknown>\n#4 0x560f94b413e0 <unknown>\n#5 0x560f94b3fa3d <unknown>\n#6 0x560f94b3f3af <unknown>\n#7 0x560f94b3f085 <unknown>\n#8 0x560f94b3ceee <unknown>\n#9 0x560f94b3d6ca <unknown>\n#10 0x560f94b5ac09 <unknown>\n#11 0x560f94bf5ed5 <unknown>\n#12 0x560f94bcee32 <unknown>\n#13 0x560f94bf52cc <unknown>\n#14 0x560f94bcec03 <unknown>\n#15 0x560f94b9b47b <unknown>\n#16 0x560f94b9c0e1 <unknown>\n#17 0x560f9507644b <unknown>\n#18 0x560f9507a37f <unknown>\n#19 0x560f9505df89 <unknown>\n#20 0x560f9507af18 <unknown>\n#21 0x560f950426df <unknown>\n#22 0x560f9509f308 <unknown>\n#23 0x560f9509f4e6 <unknown>\n#24 0x560f950b0b76 <unknown>\n#25 0x7f009da9caa4 <unknown>\n#26 0x7f009db29c3c <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/niffler-py-st3/niffler-py-st3/python_test/conftest.py\", line 90, in niffler\n    niffler.login_page.go_to_niffler()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/niffler-py-st3/niffler-py-st3/python_test/model/LoginPage.py\", line 20, in go_to_niffler\n    self.wd.get(self.base_url)\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Перейти на страницу авторизации Niffler",
      "time" : {
        "start" : 1750752652550,
        "stop" : 1750752653133,
        "duration" : 583
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=137.0.7151.103)\nStacktrace:\n#0 0x560f950b1c9a <unknown>\n#1 0x560f94b576e0 <unknown>\n#2 0x560f94b4e7e7 <unknown>\n#3 0x560f94b3f699 <unknown>\n#4 0x560f94b413e0 <unknown>\n#5 0x560f94b3fa3d <unknown>\n#6 0x560f94b3f3af <unknown>\n#7 0x560f94b3f085 <unknown>\n#8 0x560f94b3ceee <unknown>\n#9 0x560f94b3d6ca <unknown>\n#10 0x560f94b5ac09 <unknown>\n#11 0x560f94bf5ed5 <unknown>\n#12 0x560f94bcee32 <unknown>\n#13 0x560f94bf52cc <unknown>\n#14 0x560f94bcec03 <unknown>\n#15 0x560f94b9b47b <unknown>\n#16 0x560f94b9c0e1 <unknown>\n#17 0x560f9507644b <unknown>\n#18 0x560f9507a37f <unknown>\n#19 0x560f9505df89 <unknown>\n#20 0x560f9507af18 <unknown>\n#21 0x560f950426df <unknown>\n#22 0x560f9509f308 <unknown>\n#23 0x560f9509f4e6 <unknown>\n#24 0x560f950b0b76 <unknown>\n#25 0x7f009da9caa4 <unknown>\n#26 0x7f009db29c3c <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/niffler-py-st3/niffler-py-st3/python_test/model/LoginPage.py\", line 20, in go_to_niffler\n    self.wd.get(self.base_url)\n    ~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 454, in get\n    self.execute(Command.GET, {\"url\": url})\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 429, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.13.5/x64/lib/python3.13/site-packages/selenium/webdriver/remote/errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "[S]  Session Faker",
    "time" : {
      "start" : 1750752638407,
      "stop" : 1750752638472,
      "duration" : 65
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "web_driver::0",
    "time" : {
      "start" : 1750752656091,
      "stop" : 1750752656177,
      "duration" : 86
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Приложение Niffler"
  }, {
    "name" : "story",
    "value" : "Критерии поиска"
  }, {
    "name" : "feature",
    "value" : "Поиск"
  }, {
    "name" : "host",
    "value" : "fv-az1384-32"
  }, {
    "name" : "thread",
    "value" : "6349-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_app"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "date_period",
    "value" : "'today'"
  }, {
    "name" : "expected_length",
    "value" : "6"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 1,
        "skipped" : 0,
        "passed" : 1,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "110c9af0b8b8cd0e",
        "reportUrl" : "https://shade1471.github.io/niffler-py-st3/42/#testresult/110c9af0b8b8cd0e",
        "status" : "passed",
        "time" : {
          "start" : 1750735582752,
          "stop" : 1750735583719,
          "duration" : 967
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "a0c95083a2ad09fa.json",
  "parameterValues" : [ "'today'", "6" ]
}